soccer/src/soccer/strategy/agent/position/robot_factory_position.cpp

```diff
@@ -16,8 +16,63 @@ RobotFactoryPosition::RobotFactoryPosition(int r_id) : Position(r_id) {
 
 std::optional<RobotIntent> RobotFactoryPosition::get_task(WorldState& world_state,
                                                           FieldDimensions& field_dimensions) {
-    // This is where the current_position can be reassigned based on the
-    // PlayState
+    // If keeper, make no changes
+    if (robot_id_ == 0) {
```

Comment:
you can't assume 0 is the robot id. you need to get that information from the referee. If i'm not mistaken, there is already a mechanism for AAC to pass this information to its Position. if not, let's figure out how to add it.